experimental/saiportextensions.h

```diff
@@ -52,6 +52,18 @@ typedef enum _sai_port_stat_extensions_t
 {
     SAI_PORT_STAT_EXTENSIONS_RANGE_START = SAI_PORT_STAT_END,
 
+    /** DASH port LB_FAST_PATH_ICMP_IN_BYTES stat count */
+    SAI_PORT_STAT_LB_FAST_PATH_ICMP_IN_BYTES = SAI_PORT_STAT_EXTENSIONS_RANGE_START,
+
+    /** DASH port LB_FAST_PATH_ICMP_IN_PACKETS stat count */
+    SAI_PORT_STAT_LB_FAST_PATH_ICMP_IN_PACKETS,
```

Comment:
is this to count number of icmp_redirect packets, or just icmp packets??

Reply:
these counters are specific to LB fast path ICMP packets, not regular ICMP packets or regular ICMP redirect packets that we might not care, e.g. IPv6 ICMP packets with non-zero payload size.